sw/host/tests/manuf/manuf_cp_unlock_raw/src/main.rs

```diff
@@ -56,8 +55,18 @@ fn main() -> anyhow::Result<()> {
     .context("failed to transition to TEST_UNLOCKED0")?;
 
     // Check that LC state is `TEST_UNLOCKED0`.
-    let state = jtag.read_lc_ctrl_reg(&LcCtrlReg::TransitionTarget)?;
+    let state = jtag.read_lc_ctrl_reg(&LcCtrlReg::LcState)?;
     assert_eq!(state, DifLcCtrlState::TestUnlocked0.redundant_encoding());
 
     Ok(())
 }
+
+fn main() -> anyhow::Result<()> {
```

Comment:
nit: `anyhow::Result` is imported; you don't need the fully qualified path here.